README.md

```diff
@@ -12,6 +12,7 @@ controller. This means you can call them from your own (custom) buttons._
 
 Install this module using bower
 
+    bower install signature_pad --save
```

Comment:
This isn't required as it's already defined in https://github.com/legalthings/angular-signature/blob/master/bower.json#L13. If you amend that, I'll merge your PR.